### ISSUE TITLE
liquid death

### ISSUE THREAD

**Issue by chrisbecke** [Azanor/thaumcraft-suggestions#41]
Liquid death in its TC5.2 form is - to me at least - somewhat underwhelming.
Its principal feature, in which it kills mobs and yields their essence as essentia crystals, is rather pointless - in base Thaumcraft at least - as very few of the essentia produces are in any way rare via other means - or even required that much in crafting.

Also, given that essentia crystals themselves might be merged back to simply be Viz shards I propose that, instead, that Liquid Death be changed to:
1. Borrow some particle effects from the Void liquid that spills in the outer lands when you expose Starry Void blocks. For a magical fluid that kills by sucking the life out of things it looks like a pretty mundane, if appropriately purple, fluid.
2. Other than that, when it damages and kills mobs, it should as much as possible ensure that the mob drops XP, and items, as though killed by a player using a weapon with at least a Looting I enchantment.

**Comment by MalkContent** [comment]
I disagree, it shouldn't drop xp. Only player kills or appropriatly high end kill aparatuses do (golems/robots/turtles). and cheesy mobgrinder blocks. Same goes for the rare drops.  
As for special fx. I'd rather not have those, people are gonna see my traps :D

**Comment by Argis13** [comment]
In the words of the little ad kid who I can't remember what she was advertising, only this line from the ad, "Why not both?" In practice, the first liquid death on the alchemy tab that looks rather mundane, and does not "[drop] XP, and items, as though killed by a player using a weapon with at least a Looting I enchantment," and one on the Eldritch tab, the "Liquid of the Reaper," (cool name, no?) that does the same amount of damage, has all the fancy particle effects and animations, and does "[drop] XP, and items, as though killed by a player using a weapon with at least a Looting I enchantment." The first one is good for traps and initial mob killing, while the second, more advanced one is better for mob farming, but not as good for traps, which is a secondary function.
